packages/core/admin/admin/src/pages/Settings/pages/TransferTokens/EditView.tsx

```diff
@@ -263,6 +263,7 @@ const EditView = () => {
                 }}
                 token={transferToken}
                 setToken={setTransferToken}
+                canShowToken={false}
                 canEditInputs={canEditInputs}
```

Comment:
And maybe the message should be different for transfer tokens given that you
wont be able to see them again ?

We could revert back to "Make sure to copy this token, you won’t be able to see it again!"

Reply:
indeed